examples/haas.cfg

```diff
@@ -88,6 +88,10 @@ uri = sqlite:////absolute/path/to/haas.db
 # ``dry_run`` is present (regardless of its value), this functionality is
 # enabled:
 #dry_run=
+#
+# The amount of time in seconds to sleep after attempting to empty the journal 
+# in the serve_networks command.  Default in haas/cli.py is 2 seconds:
```

Comment:
We shouldn't (generally) be referencing source-locations in the config file -- remember this is an end-user visible config file -- sysadmins don't need to be expected to read the source.